src/main/resources/config/init.xml

```diff
@@ -19,8 +19,10 @@
 <template name="init" onError="fault" xmlns="http://ws.apache.org/ns/synapse">
 	<parameter name="accountName" description="The name of the azure storage account."/>
 	<parameter name="accountKey" description="The access key for the storage account."/>
+	<parameter name="defaultEndpointsProtocol" description="The protocol (http or https) of the azure storage account."/>
```

Comment:
should this be defaultEndpointProtocol?, why it is as Endpoints

Reply:
This value taken from the documentation [https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string). Otherwise we can have just `protocol`. WDYT?